vhd/packer/install-dependencies.sh

```diff
@@ -285,6 +285,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.1.1
 1.0.33
```

Comment:
let's remove 1.0.33 and 1.0.32 as well :)

Reply:
removed!